cajita/src/Cajita_FastFourierTransform.hpp

```diff
@@ -301,7 +247,7 @@ class FastFourierTransform
 //---------------------------------------------------------------------------//
 // heFFTe
 //---------------------------------------------------------------------------//
-
+// TODO: dont think need Scalar as template param for these
```

Comment:
Are you planning to do it as part of this PR?

Reply:
Removed